packages/client/components/Snackbar.tsx

```diff
@@ -57,6 +60,7 @@ const Snackbar = React.memo(() => {
   const {openPortal, terminatePortal, portal} = usePortal({id: 'snackbar', noClose: true})
   const {location} = useRouter()
   const isMeetingRoute = location.pathname.startsWith('/meet/')
+  const isStandupRoute = isMeetingRoute && !location.pathname.includes('/checkin')
```

Comment:
I think using `relay` to add dependency of `meetingType` is also an option.

Reply:
I am seeking for more clarity on this @87prashant.

Reply:
Hi @87prashant, I am not really clear on this, can you explain further?